pkg/ui/workspaces/cluster-ui/src/databaseDetailsPage/databaseDetailsPage.spec.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+import { expect } from "chai";
+import { defaultFilters } from "../queryFilter";
+import { indexUnusedDuration } from "../util";
+import * as H from "history";
+import { shallow } from "enzyme";
```

Comment:
This isn't something I expect you to change at this point, but for any new tests please use React Testing Library instead of enzyme. RTL approach tests less implementation details and the API is easier to wrap your head around. The `managed-service` repo has been fully migrated to RTL if you ever need an example of how to test something.